src/ansys/fluent/core/launcher/pim_launcher.py

```diff
@@ -141,18 +141,16 @@ def __init__(
         del argvals["self"]
         if argvals["start_timeout"] is None:
             argvals["start_timeout"] = 60
-        for arg_name, arg_values in argvals.items():
-            setattr(self, arg_name, arg_values)
         self.argvals = argvals
-        self.new_session = self.mode.value[0]
+        self.new_session = self.argvals["mode"].value[0]
 
-        if self.additional_arguments:
+        if self.argvals["additional_arguments"]:
             logger.warning(
                 "'additional_arguments' option for 'launch_fluent' is currently not supported "
                 "when starting a remote Fluent PyPIM client."
```

Comment:
```suggestion
                "'additional_arguments' option for 'launch_fluent()' method is not supported "
                "when starting a remote Fluent PyPIM client."
```
Avoid words like currently to keep documentation "timeless".

Reply:
Done. Thank you.